structlog_sentry/__init__.py

```diff
@@ -0,0 +1,45 @@
+import logging
+import os
+import sys
+
+from sentry_sdk import capture_event, utils as sentry_utils
+
+
+class SentryProcessor:
+    """Sentry processor for structlog. Uses Sentry SDK to capture events in Sentry."""
+
+    def __init__(self, sentry_log_level=logging.WARNING, active=True):
```

Comment:
Let's change `sentry_log_level` to just a `level`. It's shorter and from name of processor it's obvious that it's related to Sentry. Standard logging is also using just a `level` parameter in cases like this.